Artifacts/MsSql/Structure/Security/0040-Tables-MultipleAuthStrats.sql

```diff
@@ -0,0 +1,131 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+------------------------------ [dbo].[ResourceClaimActions] -------------------------------------------------
+CREATE TABLE [dbo].[ResourceClaimActions](
+	[ResourceClaimActionId] [int] IDENTITY(1,1) NOT NULL,
+	[ActionId] [int] NOT NULL,
+	[ResourceClaimId] [int] NOT NULL,
+	[ValidationRuleSetName] [nvarchar](255) NULL,
+ CONSTRAINT [PK_dbo.ResourceClaimActions] PRIMARY KEY CLUSTERED 
+(
+	[ResourceClaimActionId] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+-- NONCLUSTERED INDEX for [ActionId],[ResourceClaimId] columns 
+CREATE NONCLUSTERED INDEX [IX_ActionId_ResourceClaimId]
+    ON [dbo].[ResourceClaimActions]([ActionId],[ResourceClaimId] ASC);
+GO
+
+-- FOREIGN KEY for [ActionId]  column
+ALTER TABLE [dbo].[ResourceClaimActions]  WITH CHECK ADD  CONSTRAINT [FK_dbo.ResourceClaimActions_dbo.Actions_ActionId] FOREIGN KEY([ActionId])
+REFERENCES [dbo].[Actions] ([ActionId])
+ON DELETE CASCADE
+GO
+
+-- FOREIGN KEY for [[ResourceClaimId]]  column
+ALTER TABLE [dbo].[ResourceClaimActions]  WITH CHECK ADD  CONSTRAINT [FK_dbo.ResourceClaimActions_dbo.ResourceClaims_ResourceClaimId] FOREIGN KEY([ResourceClaimId])
+REFERENCES [dbo].[ResourceClaims] ([ResourceClaimId])
+ON DELETE CASCADE
+GO
+
+---------------------------- [dbo].[ClaimSetResourceClaimActions] --------------------------------------------------------
+
+CREATE TABLE [dbo].[ClaimSetResourceClaimActions](
+	[ClaimSetResourceClaimActionId] [int] IDENTITY(1,1) NOT NULL,
+	[ActionId] [int] NOT NULL,
+	[ClaimSetId] [int] NOT NULL,
+	[ResourceClaimId] [int] NOT NULL,
+	[ValidationRuleSetNameOverride] [nvarchar](255) NULL,
+ CONSTRAINT [PK_dbo.ClaimSetResourceClaimActions] PRIMARY KEY CLUSTERED 
+(
+	[ClaimSetResourceClaimActionId] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+-- NONCLUSTERED INDEX for [ActionId],[ClaimSetId],[ResourceClaimId] columns 
+CREATE NONCLUSTERED INDEX [IX_ActionId_ClaimSetId_ResourceClaimId]
```

Comment:
This should be a _unique_ constraint/index -- and the order of the columns in the index should still follow the order of columns shown above.

Reply:
I updated with CREATE UNIQUE NONCLUSTERED INDEX